### PR TITLE
Translate GA

### PR DESCRIPTION
**Pull request by lukesneeringer**
This pull request bumps Translate to GA status, and should be followed by a release.

There are a couple of last minute changes:

  * The actual surface is moved from `translate` to `translate_v2`, with a alias in `translate`.
  * Documentation on models is changed to encourage the use of a string, rather than the constants.

### CODE REVIEW

File: pubsub/nox.py [hunk 1]
```diff
@@ -34,14 +34,15 @@ def unit_tests(session, python_version):
     session.virtualenv_dirname = 'unit-' + python_version
 
     # Install all test dependencies, then install this package in-place.
-    session.install('mock', 'pytest', 'pytest-cov', *LOCAL_DEPS)
```
Comment (lukesneeringer) [inline pubsub/nox.py]: Oops, this file's presence is an error. Fixing.

File: translate/google/cloud/translate_v2/client.py [hunk 2]
```diff
@@ -189,8 +189,8 @@ def translate(self, values, target_language=None, format_=None,
                                   in the query.
 
         :type model: str
-        :param model: (Optional) The model used to translate the text. The
-                      only accepted values are :attr:`BASE` and :attr:`NMT`.
+        :param model: (Optional) The model used to translate the text, such
+                      as ``'base'`` or ``'nmt'``.
```
Comment (dhermes) [inline translate/google/cloud/translate_v2/client.py]: @lukesneeringer Are these no longer attributes? Or something else happened?

File: translate/setup.py [hunk 3]
```diff
@@ -56,7 +56,7 @@
 
 setup(
     name='google-cloud-translate',
-    version='0.25.0',
+    version='1.0.0',
```
Comment (dhermes) [inline translate/setup.py]: Plan to bump the uber-package?

File: translate/tests/system.py [hunk 4]
```diff
@@ -56,7 +56,7 @@ def test_translate(self):
         values = ['hvala ti', 'dankon',
                   'Me llamo Jeff', 'My name is Jeff']
         translations = Config.CLIENT.translate(
-            values, target_language='de', model=translate.NMT)
+            values, target_language='de', model='nmt')
```
Comment (dhermes) [inline translate/tests/system.py]: Could `translate_v2.NMT` work here or no?

File: translate/google/cloud/translate_v2/client.py (hunk 2 above)
Comment (lukesneeringer) [inline translate/google/cloud/translate_v2/client.py]: They are specified as a string in the API, and they do not want to update constants in the library whenever they add new ones.

File: translate/setup.py (hunk 3 above)
Comment (lukesneeringer) [inline translate/setup.py]: We should, yes.

File: translate/tests/system.py (hunk 4 above)
Comment (lukesneeringer) [inline translate/tests/system.py]: It will work, but it is not the format we want to advertise. Specific request from the API team.

**Pull request merged**